sdk/schemaregistry/ci.yml

```diff
@@ -0,0 +1,31 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
```

Comment:
Where did this come from? I didn't think this was auto generated.

Reply:
All the files are originally from our template, which is what we use to make generated SDKs: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/template

Reply:
But that file doesn't have this DO NOT EDIT THIS FILE comment.

Reply:
Ah I guess that was updated after you created this branch.

Reply:
It's not, it's just to scare people away from editing it.

Reply:
Well all of the CI files were updated to no longer have this comment after this branch was created, so it should be removed here as well.

Reply:
Fixed to align with the new template file's comment.